spec/data-model/README.md

```diff
@@ -48,6 +48,16 @@ declarations, options, and attributes to be optional rather than required proper
 
 ## Messages
 
```

Comment:
The note should probably be at the end of the intro, rather than within the "Messages" section.
```suggestion
```